src/Aspire.Hosting/Dcp/DcpKubernetesClient.cs

```diff
@@ -0,0 +1,112 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using k8s;
+using k8s.Autorest;
+
+namespace Aspire.Hosting.Dcp;
+
+internal class DcpKubernetesClient : k8s.Kubernetes
```

Comment:
Is there a reason for choosing inheritance here? Are required APIs protected?

Reply:
Yes. Specifically, `SendRequest`

Reply:
Ok. Could you add a comment please. That way someone won't spend time trying to refactor in the future.